src/App.tsx

```diff
@@ -21,11 +29,20 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            onChange={(event) => {
+              setPerPage(+event.target.value);
+              setCurrentPage(1);
+            }}
```

Comment:
bad practise to make arrow functions in jsx
make handler in component and pass reference here